README.md

```diff
@@ -1,2 +1,3 @@
-Cmdlet help Docs for Azure PowerShell services thats publishing on docs.microsoft.com
+## <a name="microsoft-open-source-code-of-conduct"></a>Microsoft Open Code Code of Conduct
 
+This project uses [Microsoft Open Source Code of Conduct] (https://opensource.microsoft.com/codeofconduct/). For more information, see [Code of Conduct FAQs] (https://opensource.microsoft.com/codeofconduct/faq/) or contact [opencode@microsoft.com] (mailto: opencode@microsoft.com) for further questions or comments.
```

Comment:
Formatting throughout is incorrect. Please remove the space between `](` in all links, otherwise they don't render correctly in markdown.